frontend/src/Routes.js

```diff
@@ -7,6 +7,7 @@ import Signup from "./containers/Signup";
 import StudentHome from "./containers/StudentHome";
 import InstructorHome from "./containers/InstructorHome";
 import EditSeatPlan from "./containers/EditSeatPlan";
+import Roster from "./containers/Roster";
```

Comment:
Where you're importing 'Roster' here, there actually has to be a Roster to import. Right now, Roster.js is an empty file so the site crashes when roster gets loaded. You may not have noticed because a white screen is expected anyway.

To fix this for now, just insert 'export default function Roster(){return();}' into Roster.js

Reply:
That's very informative, thank you Reid